deepchem/trans/transformers.py

```diff
@@ -1304,12 +1304,17 @@ def flip(self, direction="lr"):
           "Invalid flip command : Enter either lr (for left to right flip) or ud (for up to down flip)"
       )
 
-  def rotate(self, angle=0):
+  def rotate(self, angle=0, reshape=True, mode='constant', order=3):
     """ Rotates the image
```

Comment:
Could you convert this docstring to Numpy format. So it would look like

```
"""Rotates the image.

Parameters
-----------
angle: ...

Returns
--------
# Add Description of returned object.
"""
```

Reply:
Done :+1:

Reply:
Thanks for the fix! We still need to tweak it a bit so it's in Numpy doc style. Here's how you'd fix the angle argument:

```
angle: float (default 0)
 Denotes angle by which the image should be rotated (in Degrees). The default is 0 which corresponds to no rotation.
```
The general format is something like
```
argument: type (default value)
  Description of argument
```
Note the colon ":" is used as the separator